DOCUMENTATION.md

```diff
@@ -3010,7 +3011,7 @@ Query parameters have the following benefits:
 
 * Faster execution of similar queries. If you submit more than one query where only a value changes, the SQL service uses the
 cached query plan from the first query rather than optimizing each query again.
-* Protection against SQL injection. If you use query parameters, you don’t need to escape special characters in user-provided
+* Protection against SQL injection. Using query parameters, you don’t need to escape special characters in user-provided
```

Comment:
Let's revert this. Using X you don't need to do Y seems wrong. This is a conditional statement, we need if.